client/modules/core/__tests__/main.js

```diff
@@ -0,0 +1,83 @@
+import { Meteor } from "meteor/meteor";
```

Comment:
these tests are likely to fail on circle:ci. to get them running locally, I had to comment out the following lines in `main.js`:
``` JavaScript
 // import { localeDep } from "/client/modules/i18n";
 // import { Packages, Shops, Accounts } from "/lib/collections";
 // import { Router } from "/client/modules/router";
```